### PR TITLE
Fix a couple of bugs to do with comments on Project Requests

### PR DESCRIPTION
**Pull request by robbiecarlton**
Depends on https://github.com/Hylozoic/hylo-node/pull/208

### CODE REVIEW

File: src/reducers/posts.js [hunk 1]
```diff
@@ -109,7 +109,6 @@ export default function (state = {}, action) {
     case CREATE_COMMENT:
       return updatePostProps(state, id, {
         follower_ids: uniq((post.follower_ids || []).concat(payload.user_id)),
-        numComments: (post.numComments || 0) + 1,
```
Comment (levity) [inline src/reducers/posts.js:112]: why did this get dropped?

File: src/components/CommentSection.js [hunk 2]
```diff
@@ -26,12 +27,15 @@ export default class CommentSection extends React.Component {
   }
 
   componentDidMount () {
-    const { post: { id }, expanded } = this.props
+    const { post, post: { id }, expanded } = this.props
```
Comment (levity) [inline src/components/CommentSection.js:30]: oh *that's* how you get both the variable itself and its contents...

**Comment by robbiecarlton** [comment]
Because APPEND_COMMENT already increments the comment count, so with this
we were getting a double comment count.
I might be missing something, but it seemed like APPEND_COMMENT happens
every time CREATE_COMMENT does

**Comment by levity** [comment]
Hmm, no, it shouldn't. That appears to be another bug specific to projects. It's because we never regression tested projects when we rolled out real-time comment updates and DM's...

**Comment by levity** [comment]
Ah, I see. This is a bug specific to expanded posts, which include projects. APPEND_COMMENT can't be relied upon to increment comment count and so forth because the event listener isn't set up when the post isn't expanded.

**Comment by levity** [comment]
One fix for this would be to also send the commenter's id down with the event and then ignore any commentAdded events from yourself.

**Comment by robbiecarlton** [comment]
@levity - good now?

**Review by levity — APPROVED** [review]

**Pull request merged**